hosts/azure/binary-cache/configuration.nix

```diff
@@ -2,14 +2,16 @@
 # SPDX-License-Identifier: Apache-2.0
 {
   self,
-  config,
   pkgs,
   lib,
   ...
 }: {
-  imports = [
+  imports = lib.flatten [
```

Comment:
No need for flatten, just use `++` to concat the two lists, or eveb better not use the `with` at all:

```nix
[
  ../../azure-common.nix
  self.nixosModules.service-openssh
  self.nixosModules.service-rclone-http
]
```